klippy/extruder_auto_fan.py

```diff
@@ -0,0 +1,39 @@
+# Printer fan support
+#
+# Copyright (C) 2016  Kevin O'Connor <kevin@koconnor.net>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
+import fan
+
+class ExtruderAutoFan:
+    printer_ready=False
+    
+    def __init__(self,printer,config):
+        self.activation_temp=float(config.get('activation_temp'))
```

Comment:
This should be config.getfloat() - that way parsing errors are properly reported to the user.  I'd also recommend a sane default (eg, 50.0).